feature/signupflow/enterphone/build.gradle.kts

```diff
@@ -26,6 +27,9 @@ dependencies {
     implementation(libs.compose.ui.preview)
     debugImplementation(libs.compose.ui.tooling)
 
+    implementation(libs.amplify.core)
+    // implementation(libs.amplify.aws.auth.cognito)
```

Comment:
Why do you keep it commented instead of remove it?

Reply:
Removed